snapcraft.yaml

```diff
@@ -1,25 +1,27 @@
-name: distrobuilder
+name: lxd-imagebuilder
 base: core22
 version: git
 grade: devel
-summary: System container and VM image builder for LXC and Incus
+summary: System container and VM image builder for LXD
 description: |-
-  Distrobuilder is a tool which lets you create container and VM images for LXC and Incus.
-license: Apache-2.0
+  LXD image builder is a tool which lets you create system container and VM images for use with LXD.
```

Comment:
Maybe worth mentioning that the snap package also includes simplestream-maintainer binary